### PR TITLE
ability to use puppeteer-extra plugins like stealth

### PR DESCRIPTION
**Pull request by ganganimaulik**
#47 

 I had to reload extension on browser load for some reason when we provide puppeteer-extra package. not sure why extension is not getting detected without reloading here: https://github.com/ganganimaulik/puppeteer-stream/blob/main/src/PuppeteerStream.ts#L79

anyway it's working fine as per example added: https://github.com/ganganimaulik/puppeteer-stream/blob/feat/extra-plugin/examples/puppeteer-extra.js

### CODE REVIEW

File: extension/background.js [hunk 1]
```diff
@@ -1,6 +1,20 @@
 // @ts-nocheck
 /* global chrome, MediaRecorder, FileReader */
 
+// need to reload extension when puppeteer-extra plugin is provided
+ chrome.storage.local.get(/* String or Array */["done"], function(items){
```
Comment (Flam3rboy) [inline extension/background.js:5]: Who changes the storage to done?

Comment (ganganimaulik) [reply extension/background.js:5]: to keep track that only reloading once. 

ie: only call  			`chrome.runtime.reload ()` one time. I dont know if there's better way

**Comment by Flam3rboy** [comment]
Thank you very much for your contribution!

**Comment by ganganimaulik** [comment]
> Thank you very much for your contribution!

your welcome :)

**Comment by thomask** [comment]
[This change](https://github.com/Flam3rboy/puppeteer-stream/pull/48/files#diff-5cd2cce00d1f86fe3fd295ce934f7e65305b1e252393d943dc28f7de8a596ca5R16)  introduces a BC break

**Comment by ganganimaulik** [comment]
> [This change](https://github.com/Flam3rboy/puppeteer-stream/pull/48/files#diff-5cd2cce00d1f86fe3fd295ce934f7e65305b1e252393d943dc28f7de8a596ca5R16) introduces a BC break
> 
> image
> 
> Is this intentional?
> 
> Also `arg1` naming is not clear



> [This change](https://github.com/Flam3rboy/puppeteer-stream/pull/48/files#diff-5cd2cce00d1f86fe3fd295ce934f7e65305b1e252393d943dc28f7de8a596ca5R16) introduces a BC break
> 
> image
> 
> Is this intentional?
> 
> Also `arg1` naming is not clear

made pr to make second arg optional for ts #51 -> cant think of variable name feel free to suggest, make pr. lol

**Comment by thomask** [comment]
That was fast 😀

However (without having tested) I think the BC break is still there, because my `opts`, which were/are past as the first argument, are [not being passed](https://github.com/Flam3rboy/puppeteer-stream/pull/48/files#diff-49cf8bfbac0980717d74335cafc25734c9a4d967fd9eb836d6949bab21dbae40R92) to `puppeteer.launch`. 

Perhaps you meant to launch with the first argument in case Puppeteer is not passed as the first argument?

**Comment by ganganimaulik** [comment]
```
  const browser = await launch({
    defaultViewport: null,
    devtools: true,
  });
```

if I try to pass arguments like this, it's working fine for me - shows devtools (without new pr). not sure what you're talking about.

**Comment by thomask** [comment]
My bad, I had missed [these lines](https://github.com/Flam3rboy/puppeteer-stream/pull/48/files#diff-49cf8bfbac0980717d74335cafc25734c9a4d967fd9eb836d6949bab21dbae40R45-R49)

**Pull request merged**